app/models/education_benefits_claim.rb

```diff
@@ -37,6 +37,10 @@ def parsed_form
     @parsed_form ||= JSON.parse(form)
   end
 
+  def confirmation_number
+    "vets_gov_#{self.class.to_s.underscore}_#{id}"
```

Comment:
In an email, Shay wrote:
> The confirmation number is displayed to the user when they submit. It is just used to trace the application, if needed. (We can pull up the application in VONAPP Admin based on the confirmation number).
 
> It has no effect on the processing of the claim.

I'll ask for some followup on what "trace" means.